src/components/DropDown/DropDown.js

```diff
@@ -42,20 +43,20 @@ class DropDrown extends Component {
         ref={this.dropDrownWrapper}
         className="dropdown-wrapper">
         <ul className="l-wing">
-          <li className="dropdown-list"><a href="#">Religion</a></li>
-          <li className="dropdown-list"><a href="#">Arts</a></li>
-          <li className="dropdown-list"><a href="#">Technology</a></li>
-          <li className="dropdown-list"><a href="#">Animals</a></li>
-          <li className="dropdown-list"><a href="#">Culture</a></li>
-          <li className="dropdown-list"><a href="#">Finance</a></li>
+          <li className="dropdown-list"><Link to={'/articles/feed'}>All Articles</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Arts</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Technology</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Animals</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Culture</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Finance</Link></li>
         </ul>
         <ul className="r-wing">
-          <li className="dropdown-list"><a href="#">Sport</a></li>
-          <li className="dropdown-list"><a href="#">Medicine</a></li>
-          <li className="dropdown-list"><a href="#">Gadget</a></li>
-          <li className="dropdown-list"><a href="#">Programming</a></li>
-          <li className="dropdown-list"><a href="#">History</a></li>
-          <li className="dropdown-list"><a href="#">Fashion</a></li>
+          <li className="dropdown-list"><Link to={'#'}>Sport</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Medicine</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Gadget</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Programming</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>History</Link></li>
+          <li className="dropdown-list"><Link to={'#'}>Fashion</Link></li>
```

Comment:
I changed all the anchor tags on the dropdown to a react-router link for consistency,